src/client/components/Containers/Collapsible.tsx

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 import React, { FunctionComponent, useRef, useState, useEffect } from 'react';
 import ResizeObserver from 'resize-observer-polyfill';
-import { FlexStartColumn, ContainerProps } from './FlexContainers';
+// import { FlexStartColumn, ContainerProps } from './FlexContainers';
```

Comment:
Are all the commented out imports unnecessary? If so, delete them.

Reply:
To be clear, not just in this file - in all changed ones (and other instances you notice). We use git: we don't need to save code.